src/utils/__tests__/alder.test.ts

```diff
@@ -282,6 +285,65 @@ describe('alder-utils', () => {
     })
   })
 
+  // Tester for denne funksjonen
+  // export const getBrukerensAlderPlus1Maaned = (
+  //   person: Person,
+  //   nedreAldersgrense: Alder
+  // ): Alder => {
+  //   const brukerensAlder = person
+  //     ? transformFoedselsdatoToAlderMinus1md(person.foedselsdato)
+  //     : getAlderMinus1Maaned(nedreAldersgrense)
+  //   const beregnetMinAlder = getAlderPlus1Maaned(brukerensAlder)
+  //   return isAlderOverAnnenAlder(beregnetMinAlder, nedreAldersgrense)
+  //     ? beregnetMinAlder
+  //     : nedreAldersgrense
+  // }
+  describe('getBrukerensAlderPlus1Maaned', () => {
+    const mock_person: Person = {
```

Comment:
Du har lignende mock i json fil og i rtk queries mock. Kan du importere dem derfra for å slippe å opprette ny lokal mock her?